backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -774,6 +799,8 @@ let rec lambdaToSql
             | TUInt8 -> "smallint"
             | TInt16 -> "smallint"
             | TUInt16 -> "integer"
+            | TInt128 -> "numeric"
```

Comment:
I dont know if it's possible here, but the postgres docs advice specifying the scale here. See https://www.postgresql.org/docs/14/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL

Perhaps these should be `"numeric(0,39)"` (I don't know that 39 is the number; just whatever the right size is).

this might be better to add as a comment rather than getting bogged down on it now.